ModularTegustation/tegu_traitor_code/tegu_traitor_items.dm

```diff
@@ -69,3 +69,102 @@
 	new /obj/item/radio/headset/syndicate/alt(src)
 	new /obj/item/clothing/gloves/combat(src) //1 TC?
 	new /obj/item/gun/ballistic/automatic/pistol(src) //7 TC
+
+// Advanced hypno-flash. For psychologists.
+/obj/item/assembly/flash/hypnotic/adv
+	desc = "A modified flash device, used to beam preprogrammed orders directly into the target's mind."
+	light_color = COLOR_RED_LIGHT
+	var/hypno_cooldown = 29 SECONDS // Big damn cooldown
+	var/hypno_cooldown_current
+	var/hypno_message // User can change it by alt-clicking the flash.
+
+/obj/item/assembly/flash/hypnotic/adv/flash_carbon(mob/living/carbon/M, mob/user, power = 15, targeted = TRUE, generic_message = FALSE)
+	if(!istype(M))
+		return
+	if(user)
+		log_combat(user, M, "[targeted? "hypno-flashed(targeted)" : "hypno-flashed(AOE)"]", src)
+	else //caused by emp/remote signal
+		M.log_message("was [targeted? "hypno-flashed(targeted)" : "hypno-flashed(AOE)"]",LOG_ATTACK)
+	if(generic_message && M != user)
+		to_chat(M, "<span class='notice'>[src] emits a red, sharp light!</span>")
+	if(targeted)
+		if(M.flash_act(1, 1))
+			if(user)
+				user.visible_message("<span class='danger'>[user] blinds [M] with the flash!</span>", "<span class='danger'>You hypno-flash [M]!</span>")
+
+			if(hypno_cooldown_current > world.time) // Still on cooldown
+				to_chat(M, "<span class='hypnophrase'>The light makes you feel oddly relaxed...</span>")
+				M.add_confusion(min(M.get_confusion() + 10, 20))
+				M.dizziness += min(M.dizziness + 10, 20)
+				M.drowsyness += min(M.drowsyness + 10, 20)
+				M.apply_status_effect(STATUS_EFFECT_PACIFY, 100)
+			else
+				if(!hypno_message) // Aka default settings
+					hypno_message = "You are a secret agent, working for [user.real_name]. \
+					You must do everything they order you and never attempt to harm, or let anyone else harm them."
```

Comment:
>You must do everything they order you and never attempt to harm, or let anyone else harm them."

You must do anything they ask of you, and you must never attempt to harm them, nor allow harm to come to them.

Reply:
Done.